sdk/python/symbolchain/TypeParserBuilder.py

```diff
@@ -0,0 +1,98 @@
+import functools
+import importlib
+import operator
+
+from .TransactionDescriptorProcessor import TransactionDescriptorProcessor
+
+
+def _name_to_enum_value(mapping, enum_type, enum_value_name):
+	if enum_value_name not in mapping:
+		raise RuntimeError(f'unknown value {enum_value_name} for type {enum_type}')
+	return mapping[enum_value_name]
+
+
+def build_type_hints_map(transaction):
```

Comment:
_build_type_hints_map ?

Reply:
nope, but I'm not sure where it should be.
Both TypeParserBuilder and TransactionFactory needs it:
 * TransactionFactory - obviously
 * TypeParserBuilder  - for struct parsers